src/SelectableBox/SelectableBoxSet.jsx

```diff
@@ -18,9 +21,11 @@ const SelectableBoxSet = React.forwardRef(({
   onChange,
   type,
   columns,
+  mobileColumns,
   className,
 }, ref) => {
   const inputType = getInputType('SelectableBoxSet', type);
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
```

Comment:
[clarification] Hmm, this wouldn't allow the consumer to customize the breakpoint as implemented. What if they needed to have it switch to the `mobileColumns` at a different breakpoint than `extraSmall` for example? Similarly, I'm wondering if there would be a need to customize different column counts for different breakpoints?

for example: extra small (1 column), medium (2 columns), extra large (3 columns).

Reply:
I reworked our solution for this problem. Now it is possible to configure adaptability as flexible as possible in Selectable Box. Check, please